### ISSUE TITLE
New rule: space-around-prop-equals

### ISSUE THREAD

**Issue by WouterFlorijn** [yannickcr/eslint-plugin-react#2315]
I couldn't find a rule like this in the plugin or issues, but do tell me if it already exists.

This is a simple rule that would disallow/enforce spaces around the `=` in properties in jsx.

**Values**

`"never"`:

```jsx
// Error
<Component foo = {bar} />
<Component foo ={bar} />
<Component foo= {bar} />

// Allowed
<Component foo={bar} />
```

`"always"`:

```jsx
// Error
<Component foo={bar} />
<Component foo ={bar} />
<Component foo= {bar} />

// Allowed
<Component foo = {bar} />
```

Optionally, this rule could also have `"before"` or `"after"` values, which only allow a space before or after the equals sign (though that does seem very ugly to me).

**Comment by WouterFlorijn** [comment]
Found it.
